doxygen is reporting a couple of errors

I just ran doxygen on the source (just copy https://github.com/01org/cc-oci-runtime/blob/master/Doxyfile to the top-level for a quick test :) and it shows a couple of errors:

```
/home/james/go/src/github.com/clearcontainers/shim/src/shim.c:377: warning: argument 'header' of command @param is not found in the argument list of read_frame(struct cc_shim *shim)
/home/james/go/src/github.com/clearcontainers/shim/src/shim.c:265: warning: The following parameters of send_proxy_message(struct cc_shim *shim, uint8_t type, uint8_t opcode, const char *payload, size_t payload_len) are not documented:
  parameter 'payload_len'
```

Related to #54.

--- a/src/shim.c
+++ b/src/shim.c
@@ -269,6 +269,7 @@ write_frame(struct cc_shim *shim, struct frame *fr)
  * \param type Type of message
  * \param opcode Opcode of message type
  * \param payload Payload to be sent
+ * \param payload_len Size of \p payload
  * 
  * \return true on success, false otherwise
  */
@@ -392,7 +393,6 @@ bool read_wire_data(struct cc_shim *shim, uint8_t *buf, size_t size)
  * Read message received from proxy.
  *
  * \param shim \ref cc_shim
- * \param header \ref frame
  *
  * \return newly allocated frame on success, NULL otherwise
  */

